Library/Homebrew/Gemfile

```diff
@@ -18,6 +18,7 @@ gem "simplecov", require: false
 gem "activesupport"
 gem "concurrent-ruby"
 gem "mechanize"
+gem "patchelf"
```

Comment:
I'd suggest omitting this until we're ready to roll this out non-optionally.

Reply:
restored Gemfile and Gemfile.lock to the `master` version